README.md

```diff
@@ -397,8 +397,13 @@ The following example shows how you can use [`react-svg-loader`](https://github.
 
 ```js
 module.exports = {
-  webpack: (cfg, { dev }) => {
+  webpack: (cfg, { webpack, dev }) => {
```

Comment:
I like this core idea here. It makes totally sense.
But I'm not quite sure about the name of it.

May be it could be `webpackNpmModule`.